cluster/saltbase/salt/etcd/initd

```diff
@@ -40,6 +40,10 @@ DAEMON_USER=etcd
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets
+        # (etcd stores file per object).
```

Comment:
etcd creates segmented wal files and snapshots (not one file per key object to be clear). but more importantly, it is a server so that each active client consumes at least one file descriptor.